### ISSUE TITLE
Suppression comments are ignored when running check-contents (v0.38.0)

### ISSUE THREAD

**Issue by nwoltman** [facebook/flow#3236]
Example code:

```js
/* @flow */

// $FlowFixMe
const wrong: string = 22;
```

.flowconfig:

```
[options]
suppress_comment= \\(.\\|\n\\)*\\$FlowFixMe
suppress_comment= \\(.\\|\n\\)*\\$FlowIgnore
```

Running `flow status` or `flow check` yields `No errors!` and `Found 0 errors` respectively, but running
`flow check-contents` and passing in a file with the example code above yields:

```
-:4
  4: const wrong: string = 22;
                           ^^ number. This type is incompatible with
  4: const wrong: string = 22;
                  ^^^^^^ string


Found 1 error
```

This only started happening in v0.38.0.

**Comment by mroch** [comment]
d'oh. seems like https://github.com/facebook/flow/commit/7e1c37ad4745683de5ceed2a38c2d60b387415d5 was just completely wrong. :(

**Comment by nwoltman** [comment]
Fixed in v0.39.0
